tsconfig.json

```diff
@@ -18,6 +18,7 @@
     "noImplicitThis": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
+    "preserveSymlinks": true,
```

Comment:
This is necessary to fix this problem when running tests in watch mode

<img width="783" alt="Screenshot 2019-07-30 at 22 41 01" src="https://user-images.githubusercontent.com/1110551/62164383-9b736d00-b31c-11e9-8302-76a70fb46924.png">

[Source of solution](https://github.com/TypeStrong/ts-node/issues/831#issuecomment-492238938)